src/raven.js

```diff
@@ -1391,7 +1391,7 @@ Raven.prototype = {
             data = objectMerge({}, crumb.data);
             for (var j = 0; j < urlProps.length; ++j) {
                 urlProp = urlProps[j];
-                if (data.hasOwnProperty(urlProp)) {
+                if (data.hasOwnProperty(urlProp) && data[urlProp]) {
```

Comment:
Only problem with this is that it will skip empty strings. It's probably safer to explicitly check for undefined. Or alternatively, change `truncate` to not explode on undefined values.

Reply:
> By the way, an empty string does not need to be truncated

Good point. Okay, I mean given that there are tests, we can proceed as-is.